src/components/Redoc/Loader.tsx

```diff
@@ -48,7 +49,7 @@ export const Loader = ({ specUrl }: { specUrl: string }) => {
     scriptLoader(document, redocDependencyScript, {
       onload: () => {
         // @ts-ignore
-        Redoc.init(specUrl, options, document.getElementById('redoc-container'));
+        Redoc.init(specUrl, options, document.getElementById('redoc-container'), overrideMenuItemNavigation);
```

Comment:
ahh this is interesting, I didnt know this method

Reply:
https://redocly.com/docs/redoc/deployment/html#the-redoc-object